core/storage/classifier/gae_models.py

```diff
@@ -175,13 +183,35 @@ def create(
             interaction_id=interaction_id,
             exp_id=exp_id,
             exp_version=exp_version,
-            state_name=state_name, status=status, training_data=training_data
+            next_scheduled_check_time=next_scheduled_check_time,
+            state_name=state_name, status=status,
+            training_data=training_data,
             )
 
         training_job_instance.put()
         return instance_id
 
     @classmethod
+    def query_training_jobs(cls, cursor=None):
```

Comment:
On your dev server, can you find a way to create some jobs and then execute this method? I would expect the dev server to autogenerate an entry in index.yaml. See: https://cloud.google.com/appengine/docs/standard/python/config/indexconfig#creating_indexes_using_the_development_server

It's important that the entry is there and that it is correct, otherwise this will fail mysteriously in production. (Been there, done that :P)